misc: bump net-next vagrant box version

borkmann opened the pull request:
Add the latest net-next kernel to our CI in order to provide more test
exposure for the recently merged set [0].

  [0] https://lore.kernel.org/bpf/cover.1574452833.git.daniel@iogearbox.net/

Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cilium/cilium/9657)
<!-- Reviewable:end -->

soumynathan reviewed (commented):
Looks good to me

soumynathan reviewed (commented):
Looks like it requires a release note.

borkmann committed:
vagrant: bump development VM images

- Use bpf-next.git tree
- Use upstream bpftool again
- Removed go-bindata dependency

Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
